coderd/database/dump.sql

```diff
@@ -880,8 +880,6 @@ CREATE VIEW template_version_with_user AS
    FROM (public.template_versions
      LEFT JOIN visible_users ON ((template_versions.created_by = visible_users.id)));
 
-COMMENT ON VIEW template_version_with_user IS 'Joins in the username + avatar url of the created by user.';
-
```

Comment:
nit: don't forget to re-add the comment on the view in the up and down migrations

Reply:
good catch, thank you!